csm_web/scheduler/views/waitlistedStudent.py

```diff
@@ -17,91 +17,69 @@ def add(request, pk=None):
     """
     Endpoint: /api/waitlistedstudent/add
```

Comment:
The docstrings here haven't been updated to reflect the URL changes from `urls.py`.